http/src/main/java/io/micronaut/http/cookie/SameSite.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http.cookie;
+
+/**
+ * Possible values for the SameSite attribute.
+ * See <a href="https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-05">changes to RFC6265bis</a>
+ */
+public enum SameSite {
+  Lax,
+  Strict,
+  None;
+
+  /**
+   * Return the enum value corresponding to the passed in same-site-flag, using a case insensitive comparison.
+   *
+   * @param name value for the SameSite Attribute
+   * @return enum value for the provided name or null
+   */
+  static SameSite of(String name) {
+      if (name != null) {
+          for (SameSite each : SameSite.class.getEnumConstants()) {
+              if (each.name().equalsIgnoreCase(name)) {
+                  return each;
```

Comment:
Can you not just use `valueOf`? https://docs.oracle.com/javase/7/docs/api/java/lang/Enum.html#valueOf(java.lang.Class,%20java.lang.String)

Reply:
Removed